### PR TITLE
javascript runner can be specified when package.json has multiple runners

### PR DESCRIPTION
**Pull request by vfbiby**
Make sure these boxes are checked before submitting your pull request:

- [ ] Add fixtures and spec when implementing or updating a test runner
- [ ] Update the README accordingly
- [ ] Update the Vim documentation in `doc/test.txt`

### CODE REVIEW

File: spec/fixtures/jest/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "",
   "devDependencies": {
-    "jest": "^20.0.0"
+    "jest": "^20.0.0",
+    "mocha": "^3.3.0"
```
Comment (codeinabox) [inline spec/fixtures/jest/package.json:7]: Could you create a separate fixture folder instead called `spec/fixtures/js_multiple` as I want to keep the existing fixtures to where a single test runner is installed

File: spec/specified_js_runner_spec.vim [hunk 2]
```diff
@@ -0,0 +1,32 @@
+source spec/support/helpers.vim
+
+describe "Runner"
+
+  before
+    cd spec/fixtures/jest
```
Comment (codeinabox) [inline spec/specified_js_runner_spec.vim:6]: Update to `cd spec/fixtures/js_multiple`

**Review by codeinabox — CHANGES_REQUESTED** [review]
Thank you for this! Have left some feedback before merging this in

**Comment by codeinabox** [comment]
One other thing I remembered can you update the notes for [this part of the README](https://github.com/vim-test/vim-test#javascript) to indicate the package.json behaviour can be overridden

File: spec/fixtures/jest/package.json (hunk 1 above)
Comment (vfbiby) [reply spec/fixtures/jest/package.json:7]: good idea

File: spec/specified_js_runner_spec.vim (hunk 2 above)
Comment (vfbiby) [reply spec/specified_js_runner_spec.vim:6]: get it

**Pull request closed without merging**